__tests__/api/__utils__/seeds/users_and_follow_seeds.js

```diff
@@ -61,7 +83,7 @@ const generateUserAndFollowSeeds = async () => {
       });
     }
 
-    if (i > 30 && i <= 60) {
+    if (i > 30 && i <= 60 + 1 /** 1 = Sum of anon user */) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the condition check has been shifted by one. Ensure that this shift doesn't affect other parts of the code where this condition is used.
```diff
-    if (i > 30 && i <= 60 + 1 /** 1 = Sum of anon user */) {
+    if (i > 30 && i <= 60) {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->